docs/reference/query-dsl/span-field-masking-query.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[query-dsl-span-field-masking-query]]
+=== Span Field Masking Query
+
+Wrapper to allow span queries participate in composite single-field span queries by 'lying' about their search field. The span field masking query maps to Lucene `SpanFieldMaskingQuery`
+
+This can be used to support queries like `span-near` or `span-or` across different fields, which is not ordinarily permitted.
+
+Span field masking query is invaluable in conjunction with *multi-fields* when same content is indexed with multiple analyzers. For instance we could index a field with the standard analyzer which breaks text up into words, and again with the english analyzer which stems words into their root form. That will add more precision to queries.
+
```

Comment:
I'd remove "That will add more precision to queries" as it is pretty vague, and I can see users following the advice without understanding the reasoning.